users/forms.py

```diff
@@ -41,7 +37,7 @@ class UserLoginForm(AuthenticationForm):
     ), label='')
 
     class Meta:
-        model = User
+        model = CustomUser
```

Comment:
Generally speaking it would be better to use the [get_user_model](https://docs.djangoproject.com/en/2.1/topics/auth/customizing/#django.contrib.auth.get_user_model) helper here instead of direct import